### DISCUSSION TITLE
Suddenly only bringing up 'settings' and not the app: Windows 11

### DISCUSSION THREAD

**Discussion by ovthco** [tom-james-watson/breaktimer-app#300] [Q&A]
Windows 11: Clicking on BreakTimer in the taskbar only brings up Settings. Hovering on it does not bring up current count down time. App itself does not seem to be running? This just happened this morning. Have tried full reset and reinstalling.

1. Install BreakTimer on Windows 11
2. Open BreakTimer
3. Only brings up settings with no option to save, open BreakTimer itself

Expected to see open BreakTimer app (like I did on Mac) with current remaining time. Hovering over should show remaining time.

What should I do to get BreakTimer back? Thanks!

Windows 11 23H2 
22631.4317
Windows Feature Experience Pack 1000.22700.1041.0

![image](https://github.com/user-attachments/assets/3aef2683-d854-4332-9aeb-cb81c40f54d2)

**Comment by tom-james-watson** [comment]
I'm not sure what you mean exactly. What are you doing to "Open BreakTimer"?

The only place where it's ever said how long is left till the next break etc is by selecting the tray icon. I'm not sure if left click on the tray icon works on Windows or whether you need to specifically right click.
